scipy/signal/_peak_finding.py

```diff
@@ -10,7 +10,15 @@
 from scipy.stats import scoreatpercentile
 
 
-__all__ = ['argrelmin', 'argrelmax', 'argrelextrema', 'find_peaks_cwt']
```

Comment:
Why a permissions change on this file?

Reply:
What do you mean? I suppose you are not speaking of the new line wrapping?

Reply:
No I'm not (though I wouldn't change that either, other modules I see don't line break like this), look at the diff of this file on GitHub you'll see the permissions were changed `100644 → 100755`

Reply:
Ah, I see what you mean. That was unintentional. I must have messed that up somehow. I'll revert that as well as the line wrapping.